libfwupdplugin/fu-device.h

```diff
@@ -244,6 +244,22 @@ typedef enum {
 	 * Since: 2.0.4
 	 **/
 	FU_DEVICE_INCORPORATE_FLAG_INSTANCE_IDS = 1ull << 17,
+	/**
+	 * FU_DEVICE_INCORPORATE_FLAG_POSSIBLE_PLUGINS:
+	 *
+	 * Set the possible plugins.
+	 *
+	 * Since: 2.0.5
```

Comment:
2.0.6

Reply:
Wowsa, we do a lot of releases these days :) FIxed.